stable/nginx-ingress/templates/controller-daemonset.yaml

```diff
@@ -21,6 +21,7 @@ spec:
         component: "{{ .Values.controller.name }}"
         release: {{ .Release.Name }}
     spec:
+      serviceAccount: {{- if .Values.controller.serviceAccount }} {{ .Values.controller.serviceAccount }}{{- else }} default{{- end }}
```

Comment:
Since we don't actually _need_ a value, let's just do:
```
{{- if .Values.controller.serviceAccountName }}
  serviceAccountName: {{ .Values.controller.serviceAccountName }}
{{- end }}
```